src/main/java/com/tcoots/CodeFellowship/config/WebSecurityConfig.java

```diff
@@ -49,16 +50,26 @@ protected void configure(HttpSecurity http) throws Exception {
                 .csrf().disable()
                 .cors().disable()
                 .authorizeRequests()
-                .antMatchers("/").permitAll()
-                .antMatchers(HttpMethod.POST, "/signup").permitAll()
-                .anyRequest().authenticated()
+                    .antMatchers("/").permitAll()
+                    .antMatchers(HttpMethod.POST, "/signup").permitAll()
+                    .antMatchers("/*.css", "/", "/login", "/users/create", "/signup").permitAll()
+                    .anyRequest().authenticated()
                 .and()
-                .formLogin()
-                .loginPage("/login")
-                .permitAll()
+                    .formLogin()
+                .defaultSuccessUrl("/myprofile", true)
+                    .loginPage("/login")
+                    .permitAll()
                 .and()
-                .logout()
-                .permitAll();
+                    .logout()
+                .logoutUrl("/")
+                    .permitAll();
+    }
```

Comment:
@Override
    protected void configure(HttpSecurity http) throws Exception {
        http
                .csrf().disable()
                .cors().disable()
                .authorizeRequests()
                .antMatchers("/*.css", "/", "/login", "/users/create", "/signup").permitAll()
                .anyRequest().authenticated()
                .and()
                    .formLogin()
                    .loginPage("/login")
                .and()
                    .logout();

    }